agent/05_agent_configure.sh

```diff
@@ -32,6 +32,14 @@ function add_dns_entry {
     if ! $(sudo virsh net-dumpxml ${BAREMETAL_NETWORK_NAME} | xmllint --xpath "//dns/host[@ip = '${ip}']" - &> /dev/null); then
       sudo virsh net-update ${BAREMETAL_NETWORK_NAME} add dns-host  "<host ip='${ip}'> <hostname>${hostname}</hostname> </host>"  --live --config
     fi
+
+    # Add entries to etc/hosts for SNO IPV6 to sucessfully run the openshift conformance tests
+    if [[ $NUM_MASTERS == 1 && $IP_STACK == "v6" ]]; then
```

Comment:
Hi @pawanpinjarkar, I'm curious why only when $IP_STACK == "v6"? Are entries not needed for SNO_IPV4?

Reply:
We are only running openshift conformance test for IPV6 in case of SNO. Other configurations are compact IPV4 and HA dual stack.